src/client/ui/status-bar/index.js

```diff
@@ -321,7 +320,7 @@ export default class StatusBar extends serviceUtils.EventEmitter {
     }
 
     _bindClickOnce (elements, handler) {
-        const eventName = featureDetection.isTouchDevice ? 'touchstart' : 'mousedown';
+        const eventName = 'mousedown';
```

Comment:
We need to preserve isTouchDevice condition as is.
I think we need to research if we can add isNativeAutomation check, so in NA mode we need to bind to mousedown